homeassistant/components/androidtv/media_player.py

```diff
@@ -132,7 +132,9 @@
         vol.Optional(CONF_ADB_SERVER_IP): cv.string,
         vol.Optional(CONF_ADB_SERVER_PORT, default=DEFAULT_ADB_SERVER_PORT): cv.port,
         vol.Optional(CONF_GET_SOURCES, default=DEFAULT_GET_SOURCES): cv.boolean,
-        vol.Optional(CONF_APPS, default=dict()): vol.Schema({cv.string: cv.string}),
+        vol.Optional(CONF_APPS, default=dict()): vol.Schema(
+            {cv.string: vol.Any(cv.string, None)}
```

Comment:
If users explicitly follow the docs, this change is unnecessary because `"" != None`

Reply:
With that being said, `None` will work with the changes you have added, so it's up to you whether or not it's worth keeping.

Reply:
I’d like to keep it. I updated the example in this pull request to match the documentation pull request. I think omitting a name is more natural than using an empty string, but it’s easy to support both.

Reply:
Also, I think users would be confused why

```yaml
this.would.work: ""
this.would.not:
```

So I’d like to keep `None` for that reason. 

What do you think?